src/lib/hooks/useCurrencyLogoURIs.ts

```diff
@@ -38,6 +43,17 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
   if (networksWithUrls.includes(chainId)) {
     return `https://raw.githubusercontent.com/Uniswap/assets/master/blockchains/${networkName}/assets/${address}/logo.png`
```

Comment:
Prefer adding the asset to Uniswap/assets, instead of special-casing it?

Reply:
That was the original plan but it was decided by @cmcewen and others that we should host our own